cloud/docker/docker.py

```diff
@@ -542,7 +542,7 @@ def _human_to_bytes(number):
             return int(number[:-len(each)]) * (1024 ** i)
         i = i + 1
 
-    raise ValueError('Could not convert %s to integer' % (number,))
+    return int(number)
```

Comment:
I think this will not fix the issue, if you use e.g. ``asdfasdfkhldsaf`` as a value for ``memory_limit`` it will raise a unhandled exception ``ValueError: invalid literal for int() with base 10``.